P5/Source/Guidelines/en/ST-Infrastructure.xml

```diff
@@ -862,70 +862,85 @@ one, or vice versa, should be done with care. </p>
        </table>
          </p>
           <specGrp xml:id="DSTECAT" n="Attribute classes">
-            <!-- must come before att.ascribed.xml: -->
+            <!--
+                This specification group should contain all the
+                attribute class specifications that are part of the
+                "tei" module.
+
+                The only true requirement is that the specification of
+                a class (call it B) that is a member of another class
+                (call it A) must be specified _after_ the class of
+                which it is a member. That is, A must come before B.
+
+                But we would like the list to be as readable as
+                possible. Thus we specify the classes that are NOT
+                members of another class first, in set 1. Then, in set
+                2, we specify those classes that are a member of a
+                class in set 1. Then, in set 3, we specify those that
+                are a member of a class in set 2. (We have not needed
+                a set 4 yet, but might someday.)
+
+                Within each set classes are specified in alpahabetic
+                order.
+            -->
```

Comment:
Thanks, @sydb ! This comment is super helpful "future proofing" for editors of this page. :-)